inputFiles/singlePhaseFlow/vtk/3D_10x10x10_compressible_hex_gravity_smoke.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" ?>
+
+<Problem>
+
+  <Included>
+    <File name="./-3D_10x10x10_compressible_hex_gravity_base.xml"/>
```

Comment:
```suggestion
    <File name="./3D_10x10x10_compressible_hex_gravity_base.xml"/>
```
Is it a typo?

Reply:
Thx @jhuang2601 !
Introducing a typo while fixing a typo 😵‍💫 😉